warehouse/query-core/src/main/java/datawave/query/jexl/visitors/ExpandMultiNormalizedTerms.java

```diff
@@ -356,4 +290,20 @@ protected JexlNode expandNodeForNormalizers(JexlNode node, Object data) {
         }
         return nodeToReturn;
     }
+    
+    /**
+     * method to return if the current node is an instance of a delayed predicate
+     *
+     * @param currNode
+     * @return
+     */
+    protected boolean isDelayedPredicate(JexlNode currNode) {
```

Comment:
Looks like there are four different instances of a method with this name. Is there a need to maintain different copies? If so, what is unique about this instance?

Reply:
removed